projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -44,8 +44,6 @@ public static function on_jetpack_feature_publicize_enabled() {
 		add_action( 'rest_api_init', array( new REST_Controller(), 'register_rest_routes' ) );
 
 		add_action( 'current_screen', array( static::class, 'init_sharing_limits' ) );
-
-		self::init_social_image_generator();
```

Comment:
We might be able to do something like hook this to admin_init instead. The main issue is that the Publicize module is being initialised on frontend requests. We should probably also cache the data more because it won't change often.